### PR TITLE
Change the custom error to Ballerina error

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
Change the custom error to Ballerina error

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ function main(string... args) {
              io:println(" Successfully describe the instances : ");
              io:println(insts);
          }
-         amazonec2:AmazonEC2Error e => io:println(e);
+         amazonec2:error e => io:println(e);
```
Comment (VijithaEkanayake) [inline README.md]: This should simply be the error right?
error e => io:println(e);

File: amazonec2/amazonec2_types.bal [hunk 2]
```diff
@@ -92,85 +92,85 @@ public type AmazonEC2Connector object {
     # + minCount - The minimum number of instances to launch
     # + securityGroup - [EC2-Classic, default VPC] One or more security group names
     # + securityGroupId - One or more security group IDs
-    # + return - If success, returns EC2Instance of launched instances, else returns AmazonEC2Error object
+    # + return - If success, returns EC2Instance of launched instances, else returns error object
```
Comment (VijithaEkanayake) [inline amazonec2/amazonec2_types.bal]: Just say returns error without saying error object

File: README.md (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply README.md]: Fixed

File: amazonec2/amazonec2_types.bal (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply amazonec2/amazonec2_types.bal]: Fixed

**Review by VijithaEkanayake-zz — APPROVED** [review]

**Pull request merged**